src/models/train_model.py

```diff
@@ -1,10 +1,21 @@
 import torch
 from conv_nn import ConvNet
 from torch import nn, optim
+import hydra
+from omegaconf import OmegaConf
+import logging
 
+log = logging.getLogger(__name__)
 
-def train(epochs=20, lr=0.001):
+@hydra.main(config_path="config", config_name='default.yaml')
+
+def train(config):
     print("Training")
+    print(f"configuration: \n {OmegaConf.to_yaml(config)}")
```

Comment:
@peterampazzo had a comment about changing "print" to "log" in lines 13 to 14, right? Should it also be done in line 44? @peterampazzo